assets/js/components-noncritical.js

```diff
@@ -211,8 +211,12 @@ const componentConfiguration = [
     },
   },
   {
-    Component: SkipToMainContent,
-    selector: '#skip-to-main-content',
+    Component: AccessibilitySkips,
+    selector: '#skip-to-main-content, #skip-to-cookie-banner',
```

Comment:
since the class is written as a generic AccessibilitySkips and it's used for skip to main content as well, I think you should rename options elements to something more generic. Plus, skipCookieBanner is already a selector, it doesn't need to be in the options. I understand that you used it for checking if it's cookie, but I think it would be more clean to use selector. Something like:
  Component: AccessibilitySkips,
    selector: '#skip-to-main-content, #skip-to-cookie-banner',
    options: { 
       elementForAttributeChange: '#ng-cc'
     }
     
Actually, we can't write the initialization of that component like this, they should be separated. Because skip to main content does not observe any changes on other elements. Something like this:

{
    Component: AccessibilitySkips,
    selector: '#skip-to-main-content'
    },
    Component: AccessibilitySkips,
    selector: '#skip-to-cookie-banner',
     options: { 
       elementForAttributeChange: '#ng-cc'
     }
    }